src/ripple/shamap/Family.h

```diff
@@ -34,35 +34,42 @@ class Family
 public:
     virtual ~Family() = default;
 
```

Comment:
While we're in this area, I recommend deleting the two copy members (as you've done in the derived classes already).  This will further require explicitly defaulting the default constructor.